apps/fabric-website-resources/README.md

```diff
@@ -1,9 +1,9 @@
 # @uifabric/fabric-website-resources
 
-**Local demo app for office-ui-fabric-react controls**
+**Local demo app for @fluentui/react controls**
 
-This package contains the demo app that runs when you do `yarn start` from the root of the Fabric repo or from the `office-ui-fabric-react` package.
+This package contains the demo app that runs when you do `yarn start` from the root of the Fabric repo or from the `@fluentui/react` package.
 
 To see a live version of this app, go to https://aka.ms/fabricdemo and choose a version.
 
-See the [readme](https://github.com/OfficeDev/office-ui-fabric-react) for more information about Fabric and instructions for getting started with development, including how to run this app locally.
+See the [readme](https://github.com/Microsoft/fluentui) for more information about Fabric and instructions for getting started with development, including how to run this app locally.
```

Comment:
All the instances of github.com/microsoft should be lowercase (the org was originally uppercase and it still redirects, but it's lowercase now)